array.with().md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
Unfortunately, the issue for this entry was missing some information. You should create a new folder and rename the file to match the following path:
content/javascript/concepts/terms/arrays/terms/with/with.md

Reply:
I'm not sure what this means, could you please clarify?

Reply:
This entry has the incorrect name, and it is not saved in the appropriate location. 

The name of the file should be "with". There should not be parentheses in the file name.

The file should be saved in a folder you create named "with". That folder should be saved at -
content/javascript/concepts/terms/arrays/terms/